packages/page-template-modal/webpack.config.js

```diff
@@ -0,0 +1,15 @@
+/**
+ **** WARNING: No ES6 modules here. Not transpiled! ****
+ */
+/* eslint-disable import/no-nodejs-modules */
+
+/**
+ * External dependencies
+ */
+const getBaseWebpackConfig = require( '@automattic/calypso-build/webpack.config.js' );
+
+module.exports = () => {
+	return getBaseWebpackConfig( {
+		WP: true,
```

Comment:
this is a workaround for an issue with `calypso-build` processing scss after the webpack 5 upgrade, I'm working on a PR to demonstrate that issue.